pkg/testutil/api_check.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/pingcap/check"
+	"github.com/tikv/pd/pkg/apiutil"
+)
+
+type APICheckerUtil struct {
+	c *check.C
+}
```

Comment:
Maybe use check directly in the function?

Reply:
refactor these functions